docs/api-config.asciidoc

```diff
@@ -46,8 +46,8 @@ http(s)://{hostname}:{port}/config/v1/agents
 ==== Responses
 
 * Successful - `200`
-* {kib} endpoint is disabled - `403`
-* {kib} is unreachable - `503`
+* {es} is unreachable and {kib} endpoint is disabled - `403`
+* APM Server is starting up or both {es} and {kib} are unreachable - `503`
```

Comment:
Changed to return 503 instead of 403. This may make it harder to track down the cause of 503 in proxy since both are 503 and only the error body is different. Anyway, there will be logs in apm server. `warn` for startup and `error` for unreachable ES + no fallback fetcher.

Reply:
The logs are per fetch request, so I'm not sure if it will clutter the logs when there are a lot of incoming fetch requests from agents.